Sources/SwiftDiagnostics/DiagnosticsFormatter.swift

```diff
@@ -176,20 +176,24 @@ public struct DiagnosticsFormatter {
     return resultSourceString
   }
 
+  struct PrimaryDiagnostic {
+    var location: SourceLocation
+    var diagnostic: Diagnostic
+  }
+
```

Comment:
I don’t think you use `PrimaryDiagnostic`.

Reply:
You're right, thanks! Removed.